logging/access.go

```diff
@@ -151,7 +151,7 @@ func LogAccess(entry *AccessEntry) {
 		auditHeader = entry.Request.Header.Get(logFilter.UnverifiedAuditHeader)
 	}
 
-	accessLog.WithFields(logrus.Fields{
+	nativeData := map[string]interface{}{
```

Comment:
Can this be of type `logrus.Fields` such that we do not need to copy the nativeData in a loop?

Reply:
I was just trying to avoid any additional data overriding the builtin one. But now I think it makes sense.